etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy

```diff
@@ -103,13 +103,14 @@ class EncounterMigrator extends SqlMigrator {
 
         executeMysql("Log warnings about encounters with nonsensical encounter dates",
                 '''
-                INSERT INTO hivmigration_data_warnings (openmrs_patient_id, openmrs_encounter_id, encounter_date, field_name, field_value, warning_type)       
+                INSERT INTO hivmigration_data_warnings (openmrs_patient_id, openmrs_encounter_id, encounter_date, field_name, field_value, warning_type, flag_for_review)       
                 SELECT p.person_id as patient_id, 
                        e.encounter_id as encounter_id, 
                        e.encounter_date as encounter_date,
                        'encounter_date' as field_name,
                        e.encounter_date as field_value,
-                       'Encounter with nonsensical date' as warning_type
+                       'Encounter with nonsensical date' as warning_type,
+                       TRUE as flag_for_review
```

Comment:
By setting "TRUE" here we are basically saying "a data officer needs to look at these error at this point"... for starters, I'd think we'd just wholesale apply the flag to an warning_type...